### DISCUSSION TITLE
How to send all results to trouble.nvim?

### DISCUSSION THREAD

**Discussion by quantum-booty** [ibhagwan/fzf-lua#1109] [Q&A] [answered]
I'd like to achieve the following workflow:
1. search files
2. press \<c-t\> to send all selected items to trouble
3. close fzf-lua and open trouble
[This example ](https://github.com/ibhagwan/fzf-lua/issues/431) works for selected items, not all items.

**Comment by ibhagwan** [comment, marked as answer]
You can use the same action mentioned in #431 but also send a `select-all` commands to fzf, for example:
```lua
require'fzf-lua'.setup {
  files = {
    actions = { ['ctrl-t'] = { fn = _G.my_action, prefix = 'select-all+' } },
  }
}
```

**EDIT:** Btw, you can also invert selection with `toggle-all`, see #807, and `man fzf` and search for `AVAILABLE ACTIONS` to understand how/why this works.
